JS.md

```diff
@@ -288,10 +290,10 @@ For each [`import`](https://github.com/WebAssembly/spec/blob/master/interpreter/
 1. Let `v` be the value of performing [`Get`](http://tc39.github.io/ecma262/#sec-get-o-p)(`o`, [`i.item_name`](https://github.com/WebAssembly/spec/blob/master/interpreter/spec/ast.ml#L171))
```

Comment:
The required type of `v` differs depending on what type of import they match, so is tested in the individual cases below. In particular, for globals it _must not_ be an object.